cmd/daprd/components/binding_azure_cosmosdb.go

```diff
@@ -1,3 +1,5 @@
+//go:build all_components || stable_components
```

Comment:
Stable components would have this flag. Other flags can be added, like "aws_components"